telegram/bot.py

```diff
@@ -4007,6 +4009,14 @@ def create_chat_invite_link(
                 the connection pool).
             api_kwargs (:obj:`dict`, optional): Arbitrary keyword arguments to be passed to the
                 Telegram API.
+            name (:obj:`str`, optional): Invite link name; 0-32 characters.
+
+                .. versionadded:: 13.8
+            creates_join_request (:obj:`bool`, optional): :obj:`True`, if users joining the chat
+                via the link need to be approved by chat administrators.
+                If :obj:`True`, ``member_limit`` can't be specified.
```

Comment:
shouldn't we check this condition in the code and raise an error? Same for `edit_chat_invite_link`

Reply:
maybe not strictly necessary, but it sure is a nice touch to prevent an unecessary API call :) we do similar things in some other methods as well (although not everywhere :D)